CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
   See more in [#181](https://github.com/palkan/test-prof/issues/181).
 
+- Adds the ability to define stackprof's interval sampling by using `TEST_STACK_PROF_INTERVAL` env variable ([@LynxEyes][])
```

Comment:
Please, add your nickname to the end of the file in order for link anchor to work)

Reply:
I'm confused, wasn't that what I did? did I do it wrong?

Reply:
AH